src/savers/tvg/tvgTvgSaver.cpp

```diff
@@ -203,31 +203,28 @@ TvgBinCounter TvgSaver::serializeFill(const Fill* fill, TvgBinTag tag)
 }
 
 
-TvgBinCounter TvgSaver::serializeStroke(const Shape* shape)
+TvgBinCounter TvgSaver::serializeStroke(const Shape* shape, const Fill* fill, uint8_t color[4])
 {
     writeTag(TVG_TAG_SHAPE_STROKE);
     reserveCount();
 
+    //width
+    auto width = shape->strokeWidth();
+    auto cnt = writeTagProperty(TVG_TAG_SHAPE_STROKE_WIDTH, sizeof(width), &width);
+
     //cap
     auto flag = static_cast<TvgBinFlag>(shape->strokeCap());
-    auto cnt = writeTagProperty(TVG_TAG_SHAPE_STROKE_CAP, SIZE(TvgBinFlag), &flag);
+    if (flag != TVG_FLAG_SHAPE_STROKE_CAP_BUTT)
```

Comment:
Shouldn't be the following like TVG_TAG_FILL_FILLSPREAD?
if (flag) cnt += writeTagProperty(TVG_TAG_SHAPE_STROKE_CAP, SIZE(TvgBinFlag), &flag);

Reply:
@hermet I couldn't do it like that, since there is a problem...
The default values of the cap and join are respectively:
- in the TVG apis: `Square` (enum = 0) and `Bevel` (enum = 0)
- in the SVG standard: 'Butt' (enum = 2) and `Miter` (enum = 2)    -> source: https://www.w3.org/TR/SVG2/painting.html

I think that it's a bit late to change the order of enums, so I had to write the if statement as I did.
what do you think?

Reply:
I can't understand why you matter the svg spec in tvg format...
It just need to save/restore the tvg values from/to tvg::paint.

Reply:
I changed it, since it was wrong.

what I wanted, was to avoid saving the values, that are default in the svg format. 
so if we have a path `<path d="..." stoke-width="10" stroke="red" />` we know that this stroke will be plotted with cap=butt and join=miter.  but, since these are not the default values in ThorVG, we have to save them in the tvg file.